### PR TITLE
Add dev mode when get the html page

### PR DESCRIPTION
**Pull request by shaofengzhu**
Add dev mode when get the html page

### CODE REVIEW

File: .scripts/serve.py [hunk 1]
```diff
@@ -69,12 +76,16 @@ def do_GET(self):
             self._send_json(execute_function(Functions, data))
         elif self.path == "/functions":
             self._send_json(get_all_metadata(Functions))
-        elif self.path == "/functions.html":
+        elif self.path.startswith("/functions.html"):
```
Comment (zooba) [inline .scripts/serve.py]: You're not using any parameters on this URL, so we can go back to `==`?

File: .scripts/serve.py [hunk 2]
```diff
@@ -69,12 +76,16 @@ def do_GET(self):
...
         elif self.path == "/functions":
             self._send_json(get_all_metadata(Functions))
-        elif self.path == "/functions.html":
+        elif self.path.startswith("/functions.html"):
             self.send_response(200)
             self.send_header("Content-Type", "text/html;charset=utf-8")
-            self.send_header("Content-Length", str(len(PAGE)))
+            devMode = False
+            if bool(os.getenv("EXCEL_DEVMODE")):
+                devMode = True
```
Comment (zooba) [inline .scripts/serve.py]:
```suggestion
            devMode = os.getenv("EXCEL_DEVMODE", "").lower() in {"1", "yes", "true"}
```

File: .scripts/serve.py [hunk 3]
```diff
@@ -69,12 +76,16 @@ def do_GET(self):
...
+            pageHtml = _getPageHtml(devMode=devMode).encode()
+            self.send_header("Content-Length", str(pageHtml))
```
Comment (zooba) [inline .scripts/serve.py]:
```suggestion
            self.send_header("Content-Length", str(len(pageHtml)))

```

File: .scripts/serve.py (hunk 1 above)
Comment (shaofengzhu) [reply .scripts/serve.py]: Excel WAC will append some querystring to the URL. We need to either use urlparse to parse the real path, or just use startswith.

**Pull request merged**